sway-lib-std/src/contract_id.sw

```diff
@@ -139,6 +246,6 @@ impl ContractId {
         asm(r1: amount, r2: sub_id) {
             mint r1 r2;
         };
-        self.transfer(sha256((ContractId::from(asm() { fp: b256 }), sub_id)), amount);
+        self.transfer(AssetId::new(ContractId::from(asm() { fp: b256 }), sub_id), amount);
```

Comment:
Isn't this the `contract_id()` function?

Reply:
yes which returns a ContractId hence this file cant import it